core/store/orm/orm.go

```diff
@@ -542,12 +539,7 @@ func (orm *ORM) SetConfigValue(field string, value encoding.TextMarshaler) error
 
 // CreateJob saves a job to the database and adds IDs to associated tables.
 func (orm *ORM) CreateJob(job *models.JobSpec) error {
-	if err := orm.MustEnsureAdvisoryLock(); err != nil {
```

Comment:
some of this cleanup could be separated out into it's own PR next time to keep each PR's scope down 😊